pages/about.html

```diff
@@ -1,22 +1,32 @@
-<div class="pricing-header px-3 py-3 pt-md-5 pb-md-4 mx-auto text-center" id="about">
-    <!-- History-->
-
+<!-- about -->
+<div class="pricing-header px-3 py-3 pb-md-4 mx-auto text-center" id="about">
     <h1>History</h1>
-    <br>
-    <p class="text-justify" >LibreSprite originated as a fork of <a href="https://www.aseprite.org" rel="nofollow">Aseprite</a>, 
-    developed by <a href="https://github.com/dacap">David Capello</a>. Aseprite used to be distributed under the GNU General Public 
-    License version 2, but was moved to a proprietary license on <a href="https://github.com/aseprite/aseprite/commit/5ecc356a41c8e29977f8608d8826489d24f5fa6c">
-    August 26th, 2016</a>.</p>
-  </div>
-
-  
-<!-- Credits -->
-
+    <br />
+    <p class="text-justify">
+        LibreSprite originated as a fork of
+        <a href="https://www.aseprite.org" rel="nofollow">Aseprite</a>,
+        developed by <a href="https://github.com/dacap">David Capello</a>.
+        Aseprite used to be distributed under the GNU General Public License
+        version 2, but was moved to a proprietary license on
+        <a
+            href="https://github.com/aseprite/aseprite/commit/5ecc356a41c8e29977f8608d8826489d24f5fa6c"
+        >
+            August 26th, 2016</a
+        >.
+    </p>
+    <!-- credits -->
     <h1>Credits</h1>
-    <p class="text-justify">An <em><strong>enormous</strong></em> thank you to the original developers of 
-    <a href="https://www.aseprite.org" rel="nofollow">Aseprite</a>, without them and their original licensing 
-    this project wouldn’t exist. Additional shout-outs are given in the 
-    <a href="https://github.com/LibreSprite/LibreSprite/blob/master/CONTRIBUTORS.md">list of contributors</a>. 
-    If you have submitted work to LibreSprite and would like to have your name on the list, please create a pull 
-    request or get in touch and we’ll make it happen.</p>
-</div>
\ No newline at end of file
+    <p class="text-justify">
+        An <em><strong>enormous</strong></em> thank you to the original
+        developers of
+        <a href="https://www.aseprite.org" rel="nofollow">Aseprite</a>, without
+        them and their original licensing this project wouldn’t exist.
+        Additional shout-outs are given in the
+        <a
+            href="https://github.com/LibreSprite/LibreSprite/blob/master/CONTRIBUTORS.md"
+            >list of contributors</a
+        >. If you have submitted work to LibreSprite and would like to have your
```

Comment:
what the heck is this formatting

Reply:
it is the formatting done by Prettier (with the default preset).

Reply:
maybe i can fix that by using a `.prettierrc` file.

Reply:
check the whitespace sensitivity option